### PR TITLE
docs edit order and change index list order

### PR DESCRIPTION
**Pull request by at-sonle**
## Description
Docs Edit order
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

### CODE REVIEW

File: docs/api-references/order.md [hunk 1]
```diff
@@ -265,3 +265,74 @@ Delete order
...
+{
+    "meta": {
+        "status": "Successfully",
+        "code": 200
+    },
+    "data": {
+        "id": 26,        
+        "user_id": 1,
+        "total": 6,
+        "status": "pending",
```
Comment (at-thaitran) [inline docs/api-references/order.md]: sửa `status: 1`, thêm `status_order: pending`

File: docs/api-references/order.md [hunk 2]
```diff
@@ -265,3 +265,74 @@ Delete order
...
+                "image": "default-product.jpg",
+                "image_product_url": "http://192.168.33.10/images/products/default-product.jpg"
+            },
+            {
+                "id": 19,
+                "product_id": 1,
+                "order_id": 26,
+                "quantity": 3,
+                "price": 2,
+                "preview": "Quam fugit ratione ut ullam autem doloribus. Quis inventore saepe aut aut porro corrupti quam.",
```
Comment (at-thaitran) [inline docs/api-references/order.md]: xóa preview, vì database xóa cột preview rồi

File: docs/api-references/order.md [hunk 3]
```diff
@@ -265,3 +265,74 @@ Delete order
...
+                "name_product": "Loy Kris",
+                "image": "default-product.jpg",
+                "image_product_url": "http://192.168.33.10/images/products/default-product.jpg"
```
Comment (at-thaitran) [inline docs/api-references/order.md]: `image_product_url -> image_url`

File: docs/api-references/order.md [hunk 4]
```diff
@@ -265,3 +265,74 @@ Delete order
...
+|---|---|
+|Accept|application\json|
+|Authorization|{token_type} {access_token}|
+
+#### Query Param
+| Param | Type | Description |
+|---|---|---|
+| quantity | number | Numbers Of Product |
+| total | number | Total Order |
+| address | string | address user |
```
Comment (at-thaitran) [inline docs/api-references/order.md]: địa chỉ người nhận

**Comment by at-thaitran** [comment]
LGTM

File: docs/api-references/order.md [hunk 5]
```diff
@@ -265,3 +265,73 @@ Delete order
...
+/api/orders/{order}
+```
+Update Order
+
+#### Request Headers
+| Key | Value |
+|---|---|
+|Accept|application\json|
+|Authorization|{token_type} {access_token}|
+
```
Comment (at-quangtran) [inline docs/api-references/order.md]: request body ?

**Comment by at-thaitran** [comment]
LGTM

**Pull request merged**